backend/app/controllers/spree/admin/base_controller.rb

```diff
@@ -23,14 +23,6 @@ def authorize_admin
           authorize! action, record
         end
 
-        # Need to generate an API key for a user due to some backend actions
-        # requiring authentication to the Spree API
-        def generate_admin_api_key
```

Comment:
this appears unused, and was a red herring when trying to figure out the best way to stub